solution-base/mongodb/charts/mongodb-sharded/values.yaml

```diff
@@ -1327,12 +1351,12 @@ shardsvr:
     ##
     containerSecurityContext:
       enabled: true
-      seLinuxOptions: null
+      seLinuxOptions: {}
       runAsUser: 1001
-      runAsGroup: 0
+      runAsGroup: 1001
```

Comment:
will this not cause any issue (or at least difference) when upgrading mongo ? (like from artesca 4.0 to 4.1)